docs/src/features.md

```diff
@@ -2,11 +2,7 @@
 
 ## **Types of problems**
 
-* **qr\_mumps** can handle unsymmetric and symmetric, positive definite problems. In the first case it will use a QR factorization whereas, in the second, it will use a Cholesky factorization. In order to choose one or the other method, **qr\_mumps** must be informed about the type of the problem through the **sym** attribute of the **qrm_spmat** matrix struture: 0 means that the problem is unsymmetric and > 0 means symmetric, positive definite. Note that in the second case, only half of the matrix must be provided, i.e., if the coefficient (i, j) is provided (j, i) must not be given.
-
-## **Multithreading**
-
-* **qr\_mumps** is a parallel, multithreaded software based on the StarPU runtime system and it currently supports multicore or, more generally, shared memory multiprocessor computers. **qr\_mumps** does not run on distributed memory (e.g. clusters) parallel computers. Parallelism is achieved through a decomposition of the workload into fine-grained computational tasks which basically correspond to the execution of a BLAS or LAPACK operation on a blocks. It is strongly recommended to use sequential BLAS and LAPACK libraries and let **qr\_mumps** have full control of the parallelism. The granularity of the tasks is controlled by the **qrm\_mb** and **qrm\_nb** parameters which set the block size for partitioning internal data. Smaller values mean more parallelism; however, because this blocking factor is an upper-bound for the granularity of operations (or, more precisely for the granularity of calls to BLAS and LAPACK routines), it is recommended to choose reasonably large values in order to achieve high efficiency.
+* **qr\_mumps** can handle unsymmetric and symmetric, positive definite problems. In the first case it will use a QR factorization whereas, in the second, it will use a Cholesky factorization. In order to choose one or the other method, **qr\_mumps** must be informed about the type of the problem through the **sym** attribute of the **qrm\_spmat** matrix struture: 0 means that the problem is unsymmetric and > 0 means symmetric, positive definite. Note that in the second case, only half of the matrix must be provided, i.e., if the coefficient (i, j) is provided (j, i) must not be given.
```

Comment:
```suggestion
* **qr\_mumps** can handle unsymmetric and symmetric, positive definite problems. In the first case it will use a QR factorization whereas, in the second, it will use a Cholesky factorization. In order to choose one or the other method, **qr\_mumps** must be informed about the type of the problem through the **sym** attribute of the **qrm\_spmat** matrix struture: **false** means that the problem is unsymmetric and **true** means symmetric, positive definite. Note that in the second case, only half of the matrix must be provided, i.e., if the coefficient (i, j) is provided (j, i) must not be given.
```

Reply:
Sure!